src/style.css

```diff
@@ -1,3 +1,88 @@
+html {
+  --header-height: 60px;
+
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+}
+
+
 body {
   margin: 0;
+
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+
+  background-color: #fff;
+}
+
+.logo {
+  display: flex;
+  margin-left: 50px;
+  height: 40px;
+  width: 40px;
+}
+
+.conteiner {
+  display: flex;
+  list-style-type: none;
+  padding: 0;
+  margin: 0;
+}
+
+.link {
+  display: flex;
+  align-items: center;
+  text-decoration: none;
+  margin: 0 0 0 20px;
```

Comment:
You should avoid margin collapsing, when a child margins go through a parent and push off another parent
[useful_link](https://www.smashingmagazine.com/2019/07/margins-in-css/#parent-and-first-or-last-child-element)
So, if you want one element to be push off from another element use margins at THESE elements, not their children
And set margin-right for all elements besides last one